test/cpp/jit/test_gpu.cpp

```diff
@@ -14748,6 +14748,76 @@ TEST_F(NVFuserTest, FusionViewDtypeSameSizeOutput_CUDA) {
   testValidate(&fusion, outputs, aten_inputs, {at_x_view}, __LINE__, __FILE__);
 }
 
+#if 0
+// TODO: this is not working yet
+TEST_F(NVFuserTest, FusionViewDtypeSplittingOutput_CUDA) {
```

Comment:
Just FYI, I moved the `FusionView*` tests from `test_gpu.cpp` to `test_gpu_view.cpp`

Reply:
Thanks for the info. I will move as well.